configuration/personattributetypes/easyDocPersonAttributeTypes.csv

```diff
@@ -0,0 +1,2 @@
+Uuid,Void/Retire,Name,Description,Format,Foreign uuid,Searchable,Edit privilege,_order:1000
+29f58586-e331-4da1-a558-3d4210e48664,,impl.email,Email,java.lang.String,,,,
```

Comment:
Let's name this **additionalPersonAttributeTypes.csv**

Reply:
[Done](https://github.com/mekomsolutions/openmrs-config-easydoc/pull/4/commits/739bc2911b24e1e526238e835f2875c32df30a21)